client/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "tabWidth": 2,
```

Comment:
Wasn't sure what else to put. This is just an example width (wasn't sure between 2 or 4 🤷 )

Reply:
I think two is fine now ?